### PR TITLE
13.0 [mig] hr employee service contract

### PR DESCRIPTION
**Pull request by kranokporn**
@Saran440

### CODE REVIEW

File: hr_employee_service_contract/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "HR Employee Service from Contracts",
+    "version": "13.0.1.0.0",
+    "category": "Human Resources",
+    "website": "https://github.com/OCA/hr",
+    "author": "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (Saran440) [inline hr_employee_service_contract/__manifest__.py]:
```suggestion
    "author": "Brainbean Apps, Odoo Community Association (OCA)",
```

File: hr_employee_service_contract/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False,
+    "summary": (
+        "Employee service information & duration based on employee's" " contracts"
```
Comment (Saran440) [inline hr_employee_service_contract/__manifest__.py]:
```suggestion
        "Employee service information & duration based on employee's contracts"
```

File: hr_employee_service_contract/migrations/12.0.1.0.0/pre-migration.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
```
Comment (Saran440) [inline hr_employee_service_contract/migrations/12.0.1.0.0/pre-migration.py]: please, delete this file

ref : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration

- Remove any possible migration script from previous version.

**Review by Saran440 — COMMENTED** [review]
Code Reviewed

**Review by jarroyomorales — COMMENTED** [review]
@kranokporn Thank you for this migration!
Why have you done the changes in the tests? Just to know if its because of some reason they stopped working.

**Comment by kranokporn** [comment]
@jarroyomorales I think, the contract renewed year is overlaps with the duration of the first start contract.

**Review by jarroyomorales — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
Please cherry-pick https://github.com/OCA/hr/pull/864 fix

**Comment by jarroyomorales** [comment]
@kranokporn Hi! Can you attend this? Thank you!

> Please cherry-pick #864 fix

**Review by Saran440 — APPROVED** [review]

**Comment by Saran440** [comment]
Add search compute fields on first_contract_id and last_contract_id
I don't know why this fields is need to search. but I add search to fields.

@jarroyomorales @alexey-pelykh  Could you review again, please?

reference red travis : https://travis-ci.org/github/OCA/hr/jobs/734559415#L774

**Comment by Saran440** [comment]
@alexey-pelykh @jarroyomorales 
I add store is True in compute fields.
Is it okay or not?

**Comment by jarroyomorales** [comment]
@Saran440 LGTM :+1:

**Pull request merged**